source/Depreciation.cpp

```diff
@@ -77,32 +77,38 @@ void Depreciation::Save(DataWriter &out, int day) const
 	out.Write(NAME[isStock]);
 	out.BeginChild();
 	{
-		for(const auto &sit : ships)
-		{
-			out.Write("ship", sit.first->ModelName());
-			out.BeginChild();
+		using ShipElement = pair<const Ship *const, map<int, int>>;
+		WriteSorted(ships,
+			[](const ShipElement *lhs, const ShipElement *rhs) { return lhs->first->ModelName() < rhs->first->ModelName(); },
```

Comment:
Should we spread these functions out on multiple lines for readability? It's not as if we only have so many lines to go around.
```suggestion
			[](const ShipElement *lhs, const ShipElement *rhs)
			{
				return lhs->first->ModelName() < rhs->first->ModelName(); 
			},
```